scripts/py/gen_nextmap_maplists.py

```diff
@@ -0,0 +1,179 @@
+# Generates maplists for https://github.com/beyond-all-reason/spads_config_bar/blob/main/etc/mapLists.conf . This controls the !nextmap command in spads.
+
+import json
+import math
+
+teamsizes = ['2v2','3v3','4v4','5v5','6v6','7v7','8v8']
+ffasizes = ['ffa3','ffa4','ffa5','ffa6','ffa7','ffa8','ffa9','ffa10','ffa11','ffa12','ffa13','ffa14','ffa15','ffa16']
+teamffasizes = ['2v2v2','2v2v2v2','2v2v2v2v2','2v2v2v2v2v2','2v2v2v2v2v2v2','2v2v2v2v2v2v2v2','3v3v3','3v3v3v3','3v3v3v3v3','4v4v4','4v4v4v4','5v5v5']
+#output_string = ''
+
+def get_data(input_file):
+    with open(input_file) as f:
+        contents = json.load(f)
+    
+    teamsize_dict = {}
+    ffasize_dict = {}
+    teamffasize_dict = {}
+    certified_maps = []
+    uncertified_maps = []
+    maps_1v1 = []
+  
+    for i in teamsizes:
+        teamsize_dict[i] = []
+    for i in ffasizes:
+        ffasize_dict[i] = []
+    for i in teamffasizes:
+        teamffasize_dict[i] = []
+    
+    for i in contents:
```

Comment:
instead of doing it like that and below doing `contents[i]` you could use [`dict.values`](https://docs.python.org/3.7/library/stdtypes.html?highlight=dict%20values#dict.values) function and do:

```py
for map in contents.values():
```

Reply:
True, changed.